control/controlclient/direct.go

```diff
@@ -1695,6 +1700,54 @@ func (c *Direct) SetDeviceAttrs(ctx context.Context, attrs tailcfg.AttrUpdate) e
 	return nil
 }
 
+// SendAuditLog does a synchronous call to the control plane submit an audit log.
+//
+// Returned errors maybe be fatal or retriable. If retriable is true, the caller may
+// retry the operation at a later time.
+func (c *Auto) SendAuditLog(ctx context.Context, auditLog tailcfg.AuditLogRequest) (err error, retriable bool) {
+	return c.direct.SendAuditLog(ctx, auditLog)
+}
```

Comment:
Normally the error value itself carries the properties like whether it's retryable. See https://go.dev/wiki/ErrorValueFAQ for some background

You could instead provide/document which predicate function to use to ask the error whether it's retryable.